deployment/oss-vdb-test/k8s/gke/debian/debian-first-version.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: debian-first-version
+spec:
+  schedule: "0 1 * * *"
```

Comment:
Just wanted to confirm that this timespec is correct, as it differs from the `"0 */1 * * *"` of deployment/oss-vdb-test/k8s/gke/debian/debian-convert.yaml